devtools/gradle/src/main/java/io/quarkus/gradle/tasks/QuarkusNative.java

```diff
@@ -22,6 +22,14 @@
 import io.quarkus.bootstrap.resolver.AppModelResolver;
 import io.quarkus.bootstrap.resolver.AppModelResolverException;
 
+/**
+ * Legacy task for backwards compatibility reasons. This should not be used in new projects
+ *
+ * This has been replaced by setting quarkus.package.type=native in the configuration.
+ *
+ * @deprecated
+ */
+@Deprecated
 public class QuarkusNative extends QuarkusTask {
```

Comment:
Can we remove this class and declare the `buildNative` task to use `QuarkusBuild` with the extra parameter instead?

Reply:
Yes, we could delete this, and declare the `buildNative` as a `DefaultTask`, something like this in `QuarkusPlugin`:

    Task buildNative = tasks.create(BUILD_NATIVE_TASK_NAME, DefaultTask.class);

But this will be a breaking change instead of a deprecation. Is that ok? or should we first release the `@Deprecate` and then this update?

Reply:
If the behavior is kept the same, it's not exactly a breaking change, since no other module depends on classes of this plugin, I think it should be fine.  We should hopefully remove this task in the future, so it would be nice to show a warning if `buildNative` is invoked directly.